packages/pigeon/lib/java_generator.dart

```diff
@@ -118,95 +118,79 @@ void _writeCodec(Indent indent, Api api, Root root) {
 void _writeHostApi(Indent indent, Api api) {
   assert(api.location == ApiLocation.host);
 
-  indent.writeln(
-      '/** Generated interface from Pigeon that represents a handler of messages from Flutter.*/');
-  indent.write('public interface ${api.name} ');
-  indent.scoped('{', '}', () {
-    for (final Method method in api.methods) {
-      final String returnType = method.isAsynchronous
-          ? 'void'
+  void writeInterfaceMethod(final Method method) {
```

Comment:
Could you do a comment like the one above for the fragment that this is responsible for making?

Reply:
done